pkg/cronjob/spark_dependencies.go

```diff
@@ -68,7 +68,8 @@ func CreateSparkDependencies(jaeger *v1.Jaeger) *batchv1beta1.CronJob {
 									Image: jaeger.Spec.Storage.Dependencies.Image,
 									Name:  name,
 									// let spark job use its default values
-									Env: removeEmptyVars(envVars),
+									Env:       removeEmptyVars(envVars),
+									Resources: jaeger.Spec.Storage.Dependencies.Resources,
```

Comment:
I think other components by default use the resources specified at the root level of the spec. We should consider doing the same here.